.github/workflows/cmd.yml

```diff
@@ -0,0 +1,323 @@
+name: Command
+
+on:
+  issue_comment: # listen for comments on issues
+    types: [created]
+
+permissions: # allow the action to comment on the PR
+  contents: write
+  issues: write
+  pull-requests: write
+  actions: read
+
+jobs:
+  acknowledge:
+    if: ${{ startsWith(github.event.comment.body, '/cmd') }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add reaction to triggered comment
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.reactions.createForIssueComment({
+              comment_id: ${{ github.event.comment.id }},
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              content: 'eyes'
+            })
+
+  clean:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Clean previous comments
+        if: ${{ startsWith(github.event.comment.body, '/cmd') && contains(github.event.comment.body, '--clean') }}
+        uses: actions/github-script@v7
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            github.rest.issues.listComments({
+              issue_number: context.issue.number,
+              owner: context.repo.owner,
+              repo: context.repo.repo
+            }).then(comments => {
+              for (let comment of comments.data) {
+                console.log(comment)
+                if (
+                  ${{ github.event.comment.id }} !== comment.id &&
+                    (
+                      ((comment.body.startsWith('Command') || comment.body.startsWith('<details><summary>Command')) && comment.user.type === 'Bot') ||
+                      (comment.body.startsWith('/cmd') && comment.user.login === context.actor)
+                    )
+                ) {
+                  github.rest.issues.deleteComment({
+                    comment_id: comment.id,
+                    owner: context.repo.owner,
+                    repo: context.repo.repo
+                  })
+                }
+              }
+            })
+  help:
+    needs: clean
+    if: ${{ startsWith(github.event.comment.body, '/cmd') && contains(github.event.comment.body, '--help') }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Get command
+        uses: actions-ecosystem/action-regex-match@v2
+        id: get-pr-comment
+        with:
+          text: ${{ github.event.comment.body }}
+          regex: '^(\/cmd )(.*)$'
+
+      - name: Save output of help
+        id: help
+        env:
+          CMD: ${{ steps.get-pr-comment.outputs.group2 }} # to avoid "" around the command
+        run: |
+          echo 'help<<EOF' >> $GITHUB_OUTPUT
+          python3 .github/scripts/cmd/cmd.py $CMD >> $GITHUB_OUTPUT
```

Comment:
This line doesn't look safe, I'll write in Element.

Reply:
Improved regex which prevents passing adding arbitrary commands + check for fellows

<img width="424" alt="image" src="https://github.com/user-attachments/assets/8956e240-79bf-41d5-b1b6-132f77db6b26">